lib/assets/javascripts/turbograft/response.coffee

```diff
@@ -17,7 +17,19 @@ class TurboGraft.Response
     else
       throw new Error("Error encountered for XHR Response: #{this}")
 
+  redirectedTo: () ->
+    @xhr.getResponseHeader('X-XHR-Redirected-To')
+
+  redirectedToNewUrl: () ->
+    Boolean(
+      @redirectedTo() &&
```

Comment:
I don't see why it was weird, couldn't you just have `redirectedTo = @xhr.getResponseHeader('X-XHR-Redirected-To')` outside of this `Boolean` call and use it in place of the method call?